docs/core-concepts/core-concepts-overview.md

```diff
@@ -0,0 +1,199 @@
+---
+id: core-concepts
+title: Core Concepts
+sidebar_label: Core Concepts
+---
+
+COMIT is an open protocol facilitating cross-blockchain applications.
+This part of the documentation explains core concepts and how they relate to the COMIT protocol.
+The documentation contains embedded google slides for explaining details.
+Note that you can play/pause the slide-player and step back and forth. 
+
+More detailed information about the COMIT protocol can be found in [COMIT protocol stack](comit-protocol-stack.md).
+
+## Atomic Swaps using HTLCs
+
+[Atomic Swap](#atomic-swap)s using [HTLC](#htlc)s defines solution using time-locks for exchanging two digital assets without the need of a trusted third party.
+
+Prior to the actual swap execution there is usually a [Negotiation](#negotiation---maker-and-taker) phase where the two parties agree on the ledgers, assets and amounts to be swapped.
+
+Assuming, that Alice has ETH on Ethereum and Bob has BTC on Bitcoin, Alice wants Bob's BTC and Bob wants Alice's ETH, the execution of a swap would be like this:
+
+1. Alice's sets up by generating a secret and hashing it. She sends to Bob:
+    1. Hash of the secret. 
+    1. Her refund address on Ethereum.
+    1. Her redeem address on Bitcoin.
+    1. Proposed expiry times on the for the [HTLC](#htlc)s. (Note: That could also be done by Bob.)
+1. Bob receives the swap proposed by Alice and sends back (if he accepts):
+    1. His refund address on Bitcoin.
+    1. His redeem address on Ethereum.
+1. Alice has to fund first, she deploys an Ethereum HTLC transferring the agreed amount of ETH into the contract.
+1. Bob monitors the Ethereum ledger and notices that Alice has deployed the HTLC funding the ETH. 
+He now sends a Bitcoin transaction into the Bitcoin network that is guarded by the script-hash of the Bitcoin HTLC (Bitcoin script).
+1. Alice monitors the Bitcoin ledger and notices that Bob has sent the funding transaction. She can now redeem the Bitcoin by spending the Bitcoin by reveiling the secret.
+1. Bob is monitoring the Bitcoin ledger and notices that Alice has spent from the funding transaction by revealing the secret.
+He extracts the actual secret from Alice's redeem transaction and uses it to redeem on Ethereum. 
+
+<iframe 
+    src="https://docs.google.com/presentation/d/e/2PACX-1vRjJeto_eQaZ8k5Ct00UzceHaSs-uIuvPmTFV6jP9SP6N-EKmct8H4cfN3xa2h-RFs8Gm4Gia41Je7m/embed?start=true&loop=false&delayms=1000" 
```

Comment:
I'd suggest to replace "(2) Bob accepts swap" with "Bob sends his identities".
Also replace "actual secret" with "secret".